js/react_native/onnxruntime-react-native.podspec

```diff
@@ -19,4 +22,22 @@ Pod::Spec.new do |spec|
 
   spec.dependency "React-Core"
   spec.dependency "onnxruntime-c"
+
+  spec.xcconfig = {
+    'OTHER_CPLUSPLUSFLAGS' => '-Wall -Wextra',
+  }
+
+  if (File.exist?(File.join(root_dir, 'package.json')))
+    # Read the react native root project directory package.json file
+    root_package = JSON.parse(File.read(File.join(root_dir, 'package.json')))
+    if (root_package["onnxruntimeExtensionsEnabled"])
```

Comment:
should also update this to check for the same values as in build.gradle, i.e., "true"